uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/my-leave/components/my-leave-calendar.js

```diff
@@ -9,21 +11,352 @@ define([
     templateUrl: ['settings', function (settings) {
       return settings.pathTpl + 'components/my-leave-calendar.html';
     }],
-    controllerAs: 'calendar',
-    controller: ['$log', controller]
+    controllerAs: 'ctrl',
+    controller: ['$log', '$q', 'OptionGroup', 'AbsencePeriod', 'AbsenceType',
+      'Calendar', 'PublicHoliday', 'LeaveRequest', controller]
```

Comment:
Kept `'$log', '$q', 'OptionGroup'` first, as these are common dependencies, for the rest changed to alphabetical order.